components/minio.libsonnet

```diff
@@ -107,11 +107,11 @@
         type: s3
         config:
           bucket: thanos
-          endpoint: minio:9000
+          endpoint: %s.%s.svc.cluster.local:9000
           insecure: true
           access_key: minio
           secret_key: minio123
-      |||,
+      ||| % [ minio.config.s3namespace, minio.service.metadata.name ],
```

Comment:
I think the namespace is after the svc name

Reply:
Done